### DISCUSSION TITLE
How to create a character?

### DISCUSSION THREAD

**Discussion by geoidesic** [moo-man/WFRP4e-FoundryVTT#1480] [Q&A]
I'm having great difficulty understanding how to create a character. 

I can create a new Actor. That's fine, which gives me a character sheet... but then what?
Under `career` there is a plus button. If I click that it only gives me 4 options, all of which seem to be to do with being a mage:

![Screenshot 2023-03-17 at 16 58 13](https://user-images.githubusercontent.com/4637995/225969918-ee005a51-f905-4198-85e7-38f3f927ea57.png)

I looked on YouTube and found this: https://www.youtube.com/watch?v=GWcBRhbIIUM&lc=UgzTIWzcLUAE5ReP3jV4AaABAg.9_f393sDmsk9nMtbMoY08x

... but it seems to be using a journal which I assumed was provided by the system but from comments it seems it is a custom journal (which makes me wonder what the point of the video is).

So I'm a bit flummoxed about how to get my players to create a character?

**Comment by moo-man** [comment]
Closing this as answered in a different discussion. Once the core module has been installed the character creator will work intuitively.

Also I recommend using the discord for general Q&A as you'll get answers from other community members much more quickly.
